x-pack/plugins/actions/common/routes/connector/response/schemas/v1.ts

```diff
@@ -8,34 +8,89 @@
 import { schema } from '@kbn/config-schema';
 
 export const connectorResponseSchema = schema.object({
-  id: schema.string(),
-  name: schema.string(),
+  id: schema.string({
+    meta: {
+      description: 'The identifier for the connector.',
+    },
+  }),
+  name: schema.string({
+    meta: {
+      description: ' The name of the rule.',
+    },
+  }),
   config: schema.maybe(schema.recordOf(schema.string(), schema.any())),
-  connector_type_id: schema.string(),
-  is_missing_secrets: schema.maybe(schema.boolean()),
-  is_preconfigured: schema.boolean(),
-  is_deprecated: schema.boolean(),
-  is_system_action: schema.boolean(),
+  connector_type_id: schema.string({
+    meta: { description: 'The connector type identifier.' },
+  }),
+  is_missing_secrets: schema.maybe(
+    schema.boolean({ meta: { description: 'Indicates whether the connector is missing secrets.' } })
+  ),
+  is_preconfigured: schema.boolean({
+    meta: { description: 'Indicates whether the connector is preconfigured.' },
+  }),
+  is_deprecated: schema.boolean({
+    meta: { description: 'Indicates whether the connector is deprecated.' },
+  }),
+  is_system_action: schema.boolean({
+    meta: { description: 'Indicates whether the connector is system action.' },
+  }),
 });
 
 export const allConnectorsResponseSchema = connectorResponseSchema.extends({
-  referenced_by_count: schema.number(),
+  referenced_by_count: schema.number({
+    meta: {
+      description: ' The number of rules that are currently using the connector.',
+    },
+  }),
 });
 
 export const connectorTypesResponseSchema = schema.object({
-  id: schema.string(),
-  name: schema.string(),
-  enabled: schema.boolean(),
-  enabled_in_config: schema.boolean(),
-  enabled_in_license: schema.boolean(),
-  minimum_license_required: schema.oneOf([
-    schema.literal('basic'),
-    schema.literal('standard'),
-    schema.literal('gold'),
-    schema.literal('platinum'),
-    schema.literal('enterprise'),
-    schema.literal('trial'),
-  ]),
-  supported_feature_ids: schema.arrayOf(schema.string()),
-  is_system_action_type: schema.boolean(),
+  id: schema.string({
+    meta: {
+      description: 'The identifier for the connector.',
+    },
+  }),
+  name: schema.string({
+    meta: {
+      description: ' The name of the rule.',
+    },
+  }),
+  enabled: schema.boolean({
+    meta: {
+      description: 'Indicates whether the connector is enabled.',
+    },
+  }),
+  enabled_in_config: schema.boolean({
+    meta: {
+      description: 'Indicates whether the connector is enabled through the configs.',
```

Comment:
In case it helps make it clearer which "configs" are involved:
```suggestion
      description: 'Indicates whether the connector is enabled in the Kibana configuration.',
```